thirdparty/raknet/Rand.cpp

```diff
@@ -154,21 +154,22 @@ void seedMT( unsigned int seed, unsigned int *state, unsigned int *&next, int &l
 
 unsigned int reloadMT( unsigned int *state, unsigned int *&next, int &left )
 {
-	register unsigned int * p0 = state, *p2 = state + 2, *pM = state + M, s0, s1;
-	register int j;
+	unsigned int * p0 = state, *p2 = state + 2, *pM = state + M, s0, s1;
+    int j;
 
 	if ( left < -1 )
 		seedMT( 4357U );
 
-	left = N - 1, next = state + 1;
+    left = N - 1; next = state + 1;
 
 	for ( s0 = state[ 0 ], s1 = state[ 1 ], j = N - M + 1; --j; s0 = s1, s1 = *p2++ )
 		* p0++ = *pM++ ^ ( mixBits( s0, s1 ) >> 1 ) ^ ( loBit( s1 ) ? K : 0U );
 
 	for ( pM = state, j = M; --j; s0 = s1, s1 = *p2++ )
 		* p0++ = *pM++ ^ ( mixBits( s0, s1 ) >> 1 ) ^ ( loBit( s1 ) ? K : 0U );
 
-	s1 = state[ 0 ], *p0 = *pM ^ ( mixBits( s0, s1 ) >> 1 ) ^ ( loBit( s1 ) ? K : 0U );
+    s1 = state[ 0 ];
```

Comment:
No business being changed.

Reply:
Why does it matter that I changed it? I think this improves readability, even if it's slight.

Reply:
I try to keep the RakNet code as close to the original (one that I grabbed).

Reply:
Oh that's RakNet. My bad. I wouldn't have changed that if I realized it was from RakNet. I'll undo that.